src/js/filter.js

```diff
@@ -112,6 +112,11 @@ class Filter {
     this.onAfterUpdate()
     this.getListResults();
     setQueryParam('search', searchTerm);
+
+    gtag('event', 'search_trigger', {
```

Comment:
How do the changes here relate to #166? Are they different approaches that do the same thing? Does this catch anything else?

Reply:
I missed #166. I see that for that one we're tracking the url change and sending pageviews. That would work with the search approach as it changes the URL but it would not track the language change unless we trigger a change in URL on language change as well.

I always get lost navigating through google analytics so not sure where those results show up. I usually prefer tracking specific actions through events since I know where those are and its for the most part easy to visualize the results.